src/antares/service/api_services/run_api.py

```diff
@@ -0,0 +1,125 @@
+        super().__init__()
+        self.config = config
+        self.study_id = study_id
+        self._base_url = f"{self.config.get_host()}/api/v1"
+        self._wrapper = RequestWrapper(self.config.set_up_api_conf())
+
+    def run_antares_simulation(self, parameters: Optional[AntaresSimulationParameters] = None) -> Job:
+        url = f"{self._base_url}/launcher/run/{self.study_id}"
+        try:
+            if parameters is not None:
+                payload = parameters.to_api()
+                response = self._wrapper.post(url, json=payload)
+            else:
+                parameters = AntaresSimulationParameters()
+                response = self._wrapper.post(url)
+            job_id = response.json()["job_id"]
+            return self._get_job_from_id(job_id, parameters)
+        except APIError as e:
+            raise AntaresSimulationRunningError(self.study_id, e.message) from e
+
+    def _get_job_from_id(self, job_id: str, parameters: AntaresSimulationParameters) -> Job:
+        url = f"{self._base_url}/launcher/jobs/{job_id}"
+        response = self._wrapper.get(url)
+        job_info = response.json()
+        status = JobStatus.from_str(job_info["status"])
+        output_id = job_info["output_id"] if status == JobStatus.SUCCESS else None
+        return Job(job_id=job_id, status=status, parameters=parameters, output_id=output_id)
+
+    def wait_job_completion(self, job: Job, time_out: int) -> None:
+        start_time = time.time()
+        repeat_interval = 5
+        if job.status == JobStatus.SUCCESS:
+            self._update_job(job)
+
+        while job.status in (JobStatus.RUNNING, JobStatus.PENDING):
+            if time.time() - start_time > time_out:
+                raise SimulationTimeOutError(job.job_id, time_out)
+            time.sleep(repeat_interval)
+            self._update_job(job)
+
+        if job.status == JobStatus.FAILED or not job.output_id:
+            raise SimulationFailedError(self.study_id)
+
+        if job.parameters.unzip_output:
+            try:
+                self._wait_unzip_output(self.study_id, job, time_out)
+            except (TaskFailedError, AntaresSimulationUnzipError) as e:
+                raise SimulationFailedError(self.study_id) from e
+
+        return None
+
+    def _update_job(self, job: Job) -> None:
+        updated_job = self._get_job_from_id(job.job_id, job.parameters)
+        job.status = updated_job.status
+        job.output_id = updated_job.output_id
+
+    def _wait_unzip_output(self, ref_id: str, job: Job, time_out: int) -> None:
+        url = f"{self._base_url}/tasks"
+        repeat_interval = 2
+        payload = {"type": ["UNARCHIVE"], "ref_id": ref_id}
+        try:
+            response = self._wrapper.post(url, json=payload)
+            tasks = response.json()
+            task_id = self._get_unarchiving_task_id(job, tasks)
+            self._wait_task_completion(task_id, repeat_interval, time_out)
+        except APIError as e:
```

Comment:
I think you can simply do a
```python
except Exception as e:
    raise AntaresSimulationUnzipError(self.study_id, job.job_id, e.message) from e
```
as I think we want to raise this issue in both cases